leader-election-impl/src/main/java/com/palantir/paxos/SqliteConnections.java

```diff
@@ -41,22 +39,22 @@ private SqliteConnections() {
         // no
     }
 
-    public static DataSource getPooledDataSource(Path path) {
+    public static HikariDataSource getPooledDataSource(Path path) {
```

Comment:
discussed offline: `CloseableDataSource` might be nicer in terms of least privilege, but this is already on the classpath everywhere and we don't really want to add another layer of delegation